README.md

```diff
@@ -110,3 +110,50 @@ Future<dynamic> backgroundMessageHandler(Map<String, dynamic> message) async {
 
 ### iOS
 Make sure that you have a `NSPhotoLibraryUsageDescription` entry in your `Info.plist`.
+
+If you don't have any push notifications integrated in your application you need to:
+1. Enable push notification in you provisioning profile.
+2. Add Capability `Push Notifcations` in your mobile application.
+3. Request notifications permissions. This library allow you to request all (badge, alert, sound) at once. Just use `Intercom.requestIosNotificationPermissions()`. It will return Future<bool> if user allowed for requested permissions.
```

Comment:
```suggestion
3. Request notifications permissions. This library allows you to request all (badge, alert, sound) at once by using `Intercom.requestIosNotificationPermissions()`. It will return `Future<bool>` if user allows the requested permissions.
```